### ISSUE TITLE
Create a preparation checklist

### ISSUE THREAD

**Issue by ax-vasquez** [ax-vasquez/wedding-site#19]
We should add a user-specific section of the site to help them keep track of the things that need to be done before the wedding. This can also be used as the basis for a notification system we add later that reminds guests to make sure things are done if they haven't marked them off yet (we don't have to do this, though - just a "nice to have" to call out).
